ci-operator/config/openshift/osde2e/openshift-osde2e-main__nightly-4.14.yaml

```diff
@@ -0,0 +1,77 @@
+base_images:
+  azure:
+    name: ci-base
+    namespace: azure
+    tag: latest
+  base:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  osde2e:
+    name: osde2e
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: rosa-classic-sts
+  cron: 0 0 1 1 *
+  steps:
+    env:
+      CONFIGS: rosa,sts,stage,e2e-suite
+      INSTALL_LATEST_NIGHTLY: "4.14"
```

Comment:
It looks like this gets picked up by osde2e, but I can't see how I picks which payload to use - where is that code?

ci-operator wires in specific pullspecs into various environment variables https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release, is that being consumed by these jobs?

Reply:
I think you can assume any ci-operator triggered job (pj-rehearse, presubmits, periodics, etc) will always have RELEASE_IMAGE_LATEST set.  I was just asking to make sure if we end up creating presubmits on repos that impact you the most, it will work with the payload constructed for the PR.  It sounds like it will, but I'm not sure you really need INSTALL_LATEST_NIGHTLY

Reply:
@stbenjam Thank you for this info! In a follow up PR, I can do a test to remove setting it and verify everything works as expected with a _pj-rehearse_ like you mentioned.